0018-connector-risk-mitigations/0018-connector-risk-mitigations.md

```diff
@@ -1,10 +1,10 @@
 ---
 title: Connector Risk Mitigations
-draft: 1
+draft: 2
 ---
 # Connector Risk Mitigations
 
-Interledger connectors take some risk in exchange for the revenue they generate from facilitating payments. This document outlines the major categories of risks connectors face and suggests some possible mitigations.
+Interledger connectors take some risk in exchange for the revenue they generate from facilitating payments. This document outlines the major categories of risks connectors face and suggests some possible mitigations. Since the introduction of ILPv4, some mitigations are no longer needed.
```

Comment:
The ones that aren't needed should probably just be taken out. Having them in there is just confusing